### PR TITLE
Imx8mq se050

### PR DESCRIPTION
**Pull request by Tim-Anderson**
Add imx8mqevk and imx8mqevk with se050 to the documentation

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 31b22901f0a5deafa279c091cd1274424ea392d4 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/740/docs/artifacts/html/index.html

**Review by MatthewCroughan — APPROVED** [review]
Great work. This is perfect. It is modular and uses the kinds of `.. include::` style I think should be established for the `boards/` section, and eventually for other parts of the docs.

File: source/reference-manual/boards/imx8-flashing.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+           $ lsusb | grep NXP
+           Bus 001 Device 023: ID 1fc9:012b NXP Semiconductors i.MX 8M Dual/8M QuadLite/8M Quad Serial Downloader
+
+         .. highlight:: none
+
+         In this mode you will use the ``uuu`` tools to program the images to the eMMC.
+   
+      #. Run the command below to program the LmP to the EMMC::
+
+           $ sudo ./uuu full_image.uuu
```
Comment (mike-scott) [inline source/reference-manual/boards/imx8-flashing.rst:16]: Can we change this to:
`$ sudo mfgtool-files-<machine_name>/uuu -pp 1 mfgtool-files-<machine_name>/full_image.uuu`

File: source/reference-manual/boards/imx8-flashing.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      #. Select ``Devices by container``
+      #. Verify a device like the following:
+
+      .. figure:: /_static/boards/windows_verify.png
+          :width: 600
+          :align: center
+
+      #. Run the command below to program the LmP to the EMMC::
+
+           C:\Users\Someone> .\uuu.exe full_image.uuu
```
Comment (mike-scott) [inline source/reference-manual/boards/imx8-flashing.rst:41]: Same here:
`C:\Users\Someone> mfgtool-files-<machine_name>\uuu.exe -pp 1 mfgtool-files-<machine_name>\full_image.uuu`

**Review by mike-scott — CHANGES_REQUESTED** [review]
Couple minor changes to the command line.

The "-pp 1" parameter:
```
    -pp         usb polling period in milliseconds
```

This is needed on closed HW where the arm core will reset after a limited amount of time if a connection is not made.  Better to make this part of the normal docs.

**Comment by doanac** [comment]
Docs for 3418889a0b609c2c143793eaf1a88536f0d1981d are browesable at: https://ci.foundries.io/projects/fio-docs/builds/741/docs/artifacts/html/index.html

**Review by mike-scott — APPROVED** [review]
Most times during review, the changes would be squashed back into their respective patch.  But I'm ok with this for now.

LGTM

**Comment by doanac** [comment]
Docs for e8e3bbed2613913f63056b9b79c9ddbf655d1465 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/742/docs/artifacts/html/index.html

**Review by mike-scott — APPROVED** [review]

**Pull request merged**